modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/ChangeTrackingIndicator.js

```diff
@@ -134,16 +134,12 @@ const PublicationsSearchContainer = ({
 				return;
 			}
 
-			AUI().use('liferay-portlet-url', () => {
-				const portletURL = Liferay.PortletURL.createURL(
-					saveDisplayPreferenceURL
-				);
+			const portletURL = createPortletURL(saveDisplayPreferenceURL);
 
-				portletURL.setParameter('key', preferencesPrefix + '-' + key);
-				portletURL.setParameter('value', value.toString());
+			portletURL.setParameter('key', preferencesPrefix + '-' + key);
```

Comment:
I know you haven't changed this line apart from the indentation, but if you end up making any further changes to this file you can use template literals here.